spec/index.html

```diff
@@ -1174,23 +1174,67 @@ <h5>DELETE WHERE</h5>
-          <p><em>IRIref_from</em> specifies the IRI of a document such that a store will be able to identify, locate and read the document. The most common form will be URLs with the <em>http</em>
-          IRI schemes. Once the document has been read, the resulting triples will be inserted into the destination graph named by the IRI referred to by <em>IRIref_to</em>.</p>
-          <p>If no destination graph IRI (<em>IRIref_to</em>) is provided to load the triples into, then the data will be loaded into the default graph.</p>
-          <p>In case no RDF data can be retrieved (as opposed to the empty graph being retrieved) from the IRI denoted by <em>IRIref_from</em>, or in case the retrieval method returns an error (such
-          as, for instance an HTTP error code), the SPARQL 1.2 Update service <em class="rfc2119" title="Keyword in RFC 2119 context">SHOULD</em> return failure and the status of the Graph Store
-          <em class="rfc2119" title="Keyword in RFC 2119 context">SHOULD</em> remain in the same status as prior to the request; in case the keyword SILENT is present, however, the operation will
-          still return success and the status of the Graph Store is not specified by the present document: implementations may create the destination graph or not and partially load data, in case of
-          a transmission error where partial data has been received (which itself may be legal RDF).</p>
+
+          <p>
+            The `LOAD` operation reads an RDF document from a IRI and inserts triples
+            into the graphs of the dataset in the  <a href="#defn_graphStore">Graph Store</a>.
+          </p>
+
+          <pre class="defn nohighlight"><strong>LOAD</strong> ( <strong>SILENT</strong> )? <em><a
+            data-cite="SPARQL12-QUERY#rIRIREF">IRIref_from</a></em> ( <strong>INTO GRAPH</strong> <em><a
+            data-cite="SPARQL12-QUERY#rIRIREF">IRIref_to</a></em> )?</pre>
+
+          <p><em>IRIref_from</em> specifies the IRI of a document such that a store will be able to
+            identify, locate and read the document. The most common form of IRIs will be URLs with
+            the https and http IRI schemes.
+          </p>
+
+          <p>If a destination graph (<i>IRIref_to</i>) is specified by `INTO GRAPH`, then the
+            specified destination graph SHOULD be created if required; again, implementations
+            providing an update service over a fixed set of graphs MUST return with failure for a
+            request that would create a disallowed graph. If the destination graph already exists,
+            then no data in that graph will be removed. 
+            The RDF document is retrieved and parsed to produce an
+            <a data-cite="RDF12-CONCEPTS#dfn-rdf-graph">RDF Graph</a>.
+            This graph is
+            <a data-cite="RDF12-SEMANTICS#dfn-merging">merged</a> into the destination
+            graph in the graph store..
```

Comment:
```suggestion
          <p>If a destination graph (<i>IRIref_to</i>) is specified by `INTO GRAPH`, then the
            specified destination graph SHOULD be created if not yet present; again, implementations
            providing an update service over a fixed set of graphs MUST return with failure for a
            request that would create a disallowed graph. If the destination graph already exists,
            then no data in that graph will be removed.
            The RDF document is retrieved and parsed to produce an
            <a data-cite="RDF12-CONCEPTS#dfn-rdf-graph">RDF Graph</a>.
            This graph is
            <a data-cite="RDF12-SEMANTICS#dfn-merging">merged</a> into the destination
            graph in the graph store.
```

Reply:
This is a complicated area because of the handling of empty graphs. "required" is the SPARQL 1.1 wording and for better or worse "required" is used in various places. This is not specific to this PR.

For this PR, I'd prefer to leave the original wording to show that there is no change of behavior here.

Reply:
L1193 corrected.